docs/en/sql-reference/statements/grant.md

```diff
@@ -403,6 +403,30 @@ Allows executing [CREATE](../../sql-reference/statements/create/index.md) and [A
 
 - To delete the created table, a user needs [DROP](#drop).
 
+### CLUSTER
+
+Allows executing `ON CLUSTER` queries.
+
+```sql title="Syntax"
+GRANT CLUSTER ON *.* TO <username>
+```
+
+By default, queries with `ON CLUSTER` require the user to have the `CLUSTER` grant.
+You will get the following error if you to try to use `ON CLUSTER` in a query without first granting the `CLUSTER` privilege:
+
+```text
+Not enough privileges. To execute this query, it's necessary to have the grant CLUSTER ON *.*. 
+```
+
+The default behavior can be changed by setting the `on_cluster_queries_require_cluster_grant` setting,
+located in the `access_control_improvements` section of `config.xml` (see below), to `false`.
+
+```yaml title="config.xml"
```

Comment:
It seems weird that neither `<access_control_improvements>` nor its sub-tags, including `<on_cluster_queries_require_cluster_grant>`, are mentioned in the [server setting docs](https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings).

Also see [programs/server/config.xml](https://github.com/ClickHouse/ClickHouse/blob/master/programs/server/config.xml#L765-L802).